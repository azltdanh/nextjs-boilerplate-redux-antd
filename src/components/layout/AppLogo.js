import Link from 'next/link';
import logo from 'public/logo.svg';

const AppLogo = () => {
  return (
    <Link href="/">
      <a>
        <img src={logo} alt="Logo" />
      </a>
    </Link>
  );
};

export default AppLogo;
