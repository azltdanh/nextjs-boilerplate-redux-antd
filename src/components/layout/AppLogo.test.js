import AppLogo from './AppLogo';

describe(AppLogo.name, () => {
  it('renders', () => {
    const wrapper = shallow(<AppLogo />);
    expect(wrapper).toMatchSnapshot();
  });
});
