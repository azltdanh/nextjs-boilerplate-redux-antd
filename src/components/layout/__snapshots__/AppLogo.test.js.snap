// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AppLogo renders 1`] = `
<Link
  href="/"
>
  <a>
    <img
      alt="Logo"
      src="test-file-stub"
    />
  </a>
</Link>
`;
